tools/run_tests/python_utils/start_port_server.py

```diff
@@ -92,16 +93,16 @@ def start_port_server(port_server_port):
     waits = 0
     while True:
       if waits > 10:
-        print('killing port server due to excessive start up waits')
+        logging.warning('killing port server due to excessive start up waits')
         port_server.kill()
       if port_server.poll() is not None:
-        print('port_server failed to start')
+        logging.error('port_server failed to start')
         # try one final time: maybe another build managed to start one
         time.sleep(1)
         try:
           urllib.request.urlopen('http://localhost:%d/get' % port_server_port,
                           timeout=1).read()
-          print('last ditch attempt to contact port server succeeded')
+          logging.info('last ditch attempt to contact port server succeeded')
           break
         except:
           traceback.print_exc()
```

Comment:
(This comment thread is attached to an unchanged line with a `traceback.print_exc` call. I'm suggesting removing the `traceback.print_exc` call and placing a `logging` call where it was.)